libs/langchain-community/src/vectorstores/azure_cosmosdb.ts

```diff
@@ -205,19 +233,27 @@ export class AzureCosmosDBVectorStore extends VectorStore {
 
   /**
    * Removes specified documents from the AzureCosmosDBVectorStore.
-   * @param ids IDs of the documents to be removed. If no IDs are specified,
-   *     all documents will be removed.
+   * If no IDs or filter are specified, all documents will be removed.
+   * @param ids A list of IDs for the documents to be removed.
+   * @param filter A MongoDB filter object or list of IDs for the documents to be removed.
    * @returns A promise that resolves when the documents have been removed.
    */
-  async delete(ids?: string[]): Promise<void> {
+  async delete(
+    ids?: string | string[],
```

Comment:
Hey sorry, no I mean a single param like this so as not to break the interface:

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-community/src/vectorstores/pinecone.ts#L174

Reply:
Done, I updated and rebased the PR